docs/src/components/MigrationChanges/ForgotPasswordCallout.tsx

```diff
@@ -0,0 +1,31 @@
+import { Alert, Text } from '@aws-amplify/ui-react';
+import { useRouter } from 'next/router';
+import { Framework, AMPLIFY_5_UI_VERSIONS } from '../../data/frameworks';
+
+export const ForgotPasswordCallout = ({ framework }) => {
+  const {
+    query: { platform = 'react' },
+  } = useRouter();
+
+  if (!framework) {
+    framework = platform as Framework;
+  }
+
+  const isReactNative = framework === 'react-native';
+  const amplify5FrameworkVersion = AMPLIFY_5_UI_VERSIONS[framework];
+
+  return (
+    <Alert
+      role="none"
+      variation="info"
+      heading={`@aws-amplify/ui-${framework} v${amplify5FrameworkVersion}`}
+    >
+      <Text>
+        Use <code>resetPassword</code> in place of <code>forgotPassword</code>{' '}
+        in version{isReactNative ? '' : 's'} {amplify5FrameworkVersion}{' '}
+        {isReactNative ? '' : 'and earlier of'} of{' '}
```

Comment:
Think the resolved string here has "of of" when `isReactNative` is `false`? Example:

> Use `resetPassword` in place of `forgotPassword` in versions 5 and earlier of of `@aws-amplify/ui-react`

Reply:
Good catch